README.md

```diff
@@ -1,18 +1,51 @@
 # Polling App
 
+ [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/sarthakkundra/polling-app)
+
+ ![GitHub contributors](https://img.shields.io/github/contributors/sarthakkundra/polling-app) ![GitHub issues](https://img.shields.io/github/issues-raw/sarthakkundra/polling-app?color=sucess) ![GitHub pull requests](https://img.shields.io/github/issues-pr/sarthakkundra/polling-app?color=blueviolet)
+
+----
+        
 ## About the Project
 This is a web app similar to Kahoot. Users can post questions with multiple choices and other users can vote on it. The result of each question can be viewed in real-time in the form of charts.
 
 ## Tech Stack
-1) Node.js
-2) React.js
-3) MongoDB
-4) Ant Design
-
-### Getting Started
-1) Fork the repo.
-2) Clone the repo.
-3) Run npm install in the root directory.
-4) Run npm install in 'client' directory.
-5) Run npm run dev in the root directory
-6) App is up on port 3000.
+* [Node.js](https://nodejs.org/en/docs/)
+* [React.js](hhttps://reactjs.org/docs/getting-started.html)
+* [MongoDB](https://docs.mongodb.com/guides/)
+* [Ant Design](https://ant.design/docs/react/introduce)
+
+---
+
+## Getting Started with the project :
+### Step 1 : 
+#### Fork the repository
+<br>
+
+### Step 2 :
+#### Clone the repo. in your local computer by typing the following command in the command line
+` git clone https://github.com/sarthakkundra/polling-app.git `
```

Comment:
Add your-username instead of sarthakkundra in the command as we want people to clone their fork not this repo.